dev/com.ibm.websphere.appserver.features/visibility/public/connector-2.0/com.ibm.websphere.appserver.connector-2.0.feature

```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.connector-2.0
+visibility=public
+singleton=true
+IBM-API-Package: com.ibm.ws.jca.service; type="internal"
+IBM-ShortName: connector-2.0
+Subsystem-Name: Jakarta Connector 2.0
+-features=\
+ com.ibm.websphere.appserver.jakarta.connector-2.0,\
+ com.ibm.websphere.appserver.transaction-1.2
+-bundles=com.ibm.ws.app.manager.rar, \
+ com.ibm.ws.jca.1.7
```

Comment:
This section of the feature file I am unsure about.  I know that connector has an API level dependency on transaction.  But not sure if this is the place to specify that.  And whether or not these bundles are still necessary?

Reply:
Given the dependency, we do need include the transaction feature (which will later be swapped out to a Jakarta transaction feature).  However, I think it would be better placed within the private Jakarta Connectors feature, which would need it as well, and then this public feature would inherit it from there.